scss/civicrm/common/_replace-plain.scss

```diff
@@ -0,0 +1,8 @@
+.replace-plain {
+  @extend %custom-form-control, %open-sans-font-family;
```

Comment:
@bmnepali removed the `%open-sans-font-family` in this PR here https://github.com/compucorp/org.civicrm.shoreditch/pull/74/files#diff-dfec4b160a5f5ee7a31d577c8aee14f8R5 . Please coordinate with him